content/doc/addons/cellar.md

```diff
@@ -17,279 +16,321 @@ aliases:
 - /doc/deploy/addon/cellar
 type: docs
 ---
+{{< hextra/hero-subtitle >}}
+  Cellar is S3-compatible online file storage web service. Use it with your favorite S3 client, or download the `s3cmd` configuration file from the add-on dashboard in Clever Cloud console.
 
-Cellar is S3-compatible online file storage web service. You can use it with your favorite S3 client.
+{{< /hextra/hero-subtitle >}}
 
-To manually manage the files, you can use [s3cmd](https://s3tools.org/s3cmd). You can download a s3cmd configuration file from the add-on configuration page.
+## Creating a bucket
 
-{{< callout type="warning">}}
-  `ws-*` and `cf*` commands are not avaible with a Cellar add-on.
+Cellar stores files in buckets. When you create a Cellar add-on, no bucket exists yet.
+
+### From Clever Cloud Console
+
+{{% steps %}}
+
+#### Go to Cellar options
+
+Click on your Cellar add-on in your deployed services list to see its menu.
+
+#### Name your bucket
+
+From **Addon Dashboard**, insert the name of your bucket.
+
+{{< callout type="info">}}
+  Buckets' names are global for every region. **You can't give the same name to two different buckets in the same region**, because the URL already exists in the Cellar cluster on this region.
```

Comment:
Maybe we can add a warning here about naming: bucket names can't use underscores (`_`)